.circleci/config.yml

```diff
@@ -50,9 +50,20 @@ jobs:
       - run:
           name: Check for changes
           command: git diff --exit-code
+  check-security-configs:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    steps:
+      - checkout
+      - run:
+          name: check security configs
+          command: |
+            forge install --no-git foundry-rs/forge-std
```

Comment:
Any reason as to not just install `forge-std` as a dep and commit it? The build step should detect when its not installed and auto install it

Reply:
We probably should, since relying on forge's dependency management is simpler to maintain and safer (pinned dep versions, etc.)